### ISSUE TITLE
Meta: Transition helm to "helm classic"

### ISSUE THREAD

**Issue by krancour** [helm/helm-classic#450]
This is a meta issue to track other discreet tasks that need to be completed to:

* Change this repository's name to "helm-classic".
* Stop building a `helm` binary from this repository and _start_ building a `helmc` binary instead.
    * This is a tactical move to create a more obvious distinction between this repository (and the tool built from it) and the [kubernetes/helm](https://github.com/kubernetes/helm) repository (and the next generation `helm` tool built from that).  This is intended to start addressing questions like that posed in #427.

Discrete tasks:

Helm tasks - pre-release:

- [x] Change the name of this repository to "helm-classic".  ~~(This should probably come last.)~~ #449 
    - This can happen at any time since we're using a containerized build.  IMO, it makes the most sense to   do it after we have merged updated docs, package names, home, and binary.
- [x] Rename packages. #451 
- [x] Update helm documentation. #452
- [x] Update build to produce `helmc` binary. #453
    - Will likely require changes in bintray as well.
    - Ensure generators in existing charts that recursively invoke the `helm` binary, somehow invoke `helmc` instead _without_ requiring modifications to charts.  i.e. Existing charts must not break when a user switches to `helmc`.
- [x] ~~_Maybe_~~ update `HELM_HOME` env var to `HELMC_HOME` and its default value from `~/.helm` to `~/.helmc`.  This is largely dependent on how safely `helmc` and the new `helm` can or cannot share a common home.  ~~Ping @technosophos.  Thoughts?~~ Per @michelleN, we _will_ do this. #459 
- [x] Update interactive help for `helmc`. #469 
- [x] Add migration docs-- explain how to get from "original" Helm to Helm Classic and how Helm Classic can co-exist with kubernetes/helm for as long as needed going forward.
    - This works because they use different home dirs.
    - Migration from original Helm should address moving / renaming existing home dir to clear the way for the new Helm.  (e.g. `mv $(helm home) $(helmc home) && rm $(which helm)`) #463
- [x] Release Helm Classic 0.7.0

Helm tasks - post-release:

- [x] Update Helm Classic change log
- [x] Update / deploy install script. https://github.com/engineyard/get-helm/pull/1
- [x] Update https://helm.sh. https://github.com/helm/helm-www/pull/21

Other related tasks:

- [x] Update [deis/workflow](https://github.com/deis/workflow) documentation to direct users to install via `helmc`. https://github.com/deis/workflow/issues/211
- [x] Update [sgoings/chart-mate](https://github.com/sgoings/chart-mate) to use `helmc`. https://github.com/sgoings/chart-mate/issues/81

We should add to these lists as needed.

cc @kmala and @sgoings.  Please feel free to amend if you feel I have missed anything.

**Comment by technosophos** [comment]
On the k8s/helm project, we've been trying to figure out how best to deal with the HELM_HOME thing, too. I'm gonna bring in @michelleN, since she's the one leading that.

**Comment by michelleN** [comment]
@krancour +1 on updating HELM_HOME --> HELMC_HOME and changing its default value to ~/.helmc.
We're using ~/.helm as HELM_HOME for kubernetes/helm but were thinking about the possible conflicts with helm classic. This change would be great. 👍

**Comment by krancour** [comment]
Thanks @michelleN.  We'll do that then.

**Comment by slack** [comment]
I think of this in three phases.

**Phase 1: Today**

https://helm.sh stays installs helm-classic

**Phase 2: when k8s/helm is viable**

* Update https://helm.sh to point to k8s/helm
* Provide information about the transition from classic to k8s/helm
* Installation instructions for each 

**Phase 3: the future**

After k8s/helm has been out in the wild 4-6 months out, pull down all helm-classic stuff.

@kmala @krancour seem reasonable enough?

**Comment by krancour** [comment]
@slack "Helm Classic," however, is a _new_, differently named binary (`helmc`) meant to pave the way for a period of overlap where the old tool may still be used in parallel with the new.  So, I'm thinking the install script and web site may require minor updates once we release Helm Classic.

**Comment by slack** [comment]
Yeah, it makes sense to update helm.sh on release of Helm Classic. But we shouldn't put together a net-new web property, I want to keep the eyes on helm.sh through the transition period.

**Comment by krancour** [comment]
@slack I'm thinking the website needs something like: "Helm comes in two flavors!  To install Helm Classic (new name, same great taste?), yada, yada, yada.  Stay tuned for the new k8s/helm, coming soon!"

**Comment by rimusz** [comment]
+1 for @krancour suggestion

**Comment by krancour** [comment]
@slack @jackfrancis How can the install script be updated (when the time comes)?  I have a feeling it's just chilling in an s3 site somewhere.

**Comment by kmala** [comment]
@krancour  i had already done that https://github.com/engineyard/get-helm/pull/1
